.github/workflows/update_lists.yml

```
@@ -52,9 +52,9 @@ jobs:
           git config --local user.email '46617994+touhidurrr@users.noreply.github.com'
           git config --local user.name 'touhidurrr'
           git add ipv?_list.txt README.md
-          git commit -m 'Updated Lists [Automated]'
+          git diff --quiet || git commit -m "Updated Lists [Automated]"
```

```suggestion
          git commit -m 'Updated Lists [Automated]'
```

This is intended. I intend the action to fail if there is nothing new to commit. This way it is easier to check how many actions have failed and how many succeeded .

---

got it